.golangci.yml

```diff
@@ -116,10 +116,12 @@ linters:
     - gosimple
     - interfacer
     - lll
+    - maligned
```

Comment:
Just a thought: this linter makes future cherry-picking harder. Git is not good at detecting reordering of lines so it will not be able merge changes easily.